static/src/stylesheets/module/content/_photo-essay.scss

```diff
@@ -0,0 +1,93 @@
+.content--article {
+    .element-image--photo-esssay {
+        padding-bottom: 0;
+        margin-bottom: 0.5rem;
+
+        &.fig--no-caption.fig--has-shares {
+            padding-bottom: 0px !important;
+        }
+
+        .block-share {
+            display: none !important;
```

Comment:
> these get given !important else where so they need to have !important here

😱 ... I suppose this is where I ask how far down the rabbit hole do we go? Is it possible to remove some of the specificity (or `!important`) from elsewhere, so that we don't have to use `!important` to override?

Unless it is `!importants` all the way down... 🐢🐢🐢🐢🐢🐢🐢🐢